Packs/DNSDB/Integrations/DNSDB_v2/DNSDB_v2.py

```diff
@@ -239,13 +301,48 @@ def _query(self, path: str, limit: int = None, time_first_before: timeval = None
             )
         )
 
-        res = self._http_request('GET', path, params=params, stream=True, resp_type='response')
+        res = self._http_request('GET', path,
+                                 params=params,
+                                 stream=True, resp_type='response')
 
         if res.status_code == 404:
+            raise DemistoException('http 404')
```

Comment:
As you removed the 404 from the OK codes of the client, I believe this is not needed anymore.

Reply:
54cd28430b